opm/simulators/flow/FlowMainEbos.hpp

```diff
@@ -234,6 +234,20 @@ namespace Opm
 #endif
                 if ( unknownKeyWords )
                 {
+                    if ( output_cout_ )
+                    {
+                        std::string msg = "Aborting simulation due unknown "
```

Comment:
Sorry to add another minor thing: it should be "due to unknown"...

Another thing that I noticed only now, but is not added in this PR: we are using exit() to terminate here (line 254), which kills the program without cleaning up properly. I think `return EXIT_FAILURE` would be better. Could you fix that as well? (If you agree.)